corehq/apps/userreports/es/adapter.py

```diff
@@ -1,14 +1,83 @@
+import datetime
 from elasticsearch import NotFoundError
 from corehq.apps.userreports.util import get_table_name
 from corehq.apps.userreports.adapter import IndicatorAdapter
+from corehq.apps.es.es_query import HQESQuery
 from corehq.elastic import get_es_new
+from dimagi.utils.decorators.memoized import memoized
 from pillowtop.es_utils import INDEX_STANDARD_SETTINGS
 
 
 def normalize_id(values):
     return '-'.join(values).replace(' ', '_')
 
 
+class ESAlchemyResults(object):
+
+    def __init__(self, keys, values=dict):
```

Comment:
I don't think the default value should be `dict` which is a callable. `None` is probably what you want.